cmis/models/cmis_binding.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class CmisBinding(models.AbstractModel):
+    _name = 'cmis.binding'
+    _inherit = 'external.binding'
+    _description = 'DMS Binding (Abstract)'
+
+    cmis_backend_id = fields.Many2one(
+        'cmis.backend', 'CMIS Backend', required=True,
+        ondelete='restrict', oldname='backend_id')
```

Comment:
@lmignon the field is in your binding table, not in the table of the model itself. So there is no conflict from having multiple bindings for the same model. Each binding is a different model.

So your magento binding and your cmis binding can coexist with the same field name backend_id.